Packs/CiscoASA/README.md

```diff
@@ -12,3 +12,13 @@ You can configure the specific vendor and product for this instance.
 3. When configuring the Syslog Collector, set:
    - vendor as vendor<- Cisco
    - product as product<- ASA
+
+### config timeformat on Cisco ASA
+The only supported date format is RFC 5424, an example: "2023-04-09T16:30:00Z"
```

Comment:
Please add a comment that if the user will use different timestamp formant time extraction and mapping will not be supported.

Reply:
Add the following:
Note: If a different timestamp format is used, time extraction and mapping will not be supported.